### PR TITLE
Updating installation docs for GA

### PR DESCRIPTION
**Pull request by abrennan89**

### CODE REVIEW

File: modules/serverless-uninstalling-knative-serving.adoc [hunk 1]
```diff
@@ -5,23 +5,33 @@
...
+After uninstalling the {ServerlessOperatorName}, the Operator CRDs and API services remain on the cluster. Use this procedure to completely uninstall the remaining components.
+
+.Prerequisite
+*  You have uninstalled Knative Serving and removed the {ServerlessOperatorName} using the previous procedure.
+
+.Procedure
+. Delete the remaining Knative Serving CRDs:
++
+----
+$ oc delete crd knativeserving.operator.knative.dev
```
Comment (maschmid) [inline modules/serverless-uninstalling-knative-serving.adoc]: Should be `knativeservings.operator.knative.dev`   (note the plural)

File: serverless/installing_serverless/removing-openshift-serverless.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+include::modules/common-attributes.adoc[]
+:context: removing-openshift-serverless
+
+toc::[]
+
+This guide provides details of how to remove the {ServerlessOperatorName}, as well as other {ServerlessProductName} components.
+
+== Removing the {ServerlessOperatorName}
+
+Before you can remove other {ServerlessProductName} components, such as Knative Serving or Eventing, you must remove the {ServerlessOperatorName}.
```
Comment (maschmid) [inline serverless/installing_serverless/removing-openshift-serverless.adoc]: I think it should be the other way around? (You should delete KnativeServing first, then the serverless-operator.)

**Comment by maschmid** [comment]
QE approved

**Comment by jboxman** [comment]
@abrennan89, I think this was introduced somewhere:

```
Building OpenShift Container Platform for branch 'abrennan89-install'
asciidoctor: ERROR: <stdin>: line 24: level 0 sections can only be used when doctype is book
```

File: modules/serverless-uninstalling-knative-serving.adoc [hunk 3]
```diff
@@ -1,27 +1,22 @@
...
-. To remove Knative Serving, use the following command:
+. Remove Knative Serving:
 +
 ----
-$ oc delete knativeserving.operator.knative.dev knative-serving -n knative-serving
+$ oc delete knativeservings.operator.knative.dev knative-serving -n knative-serving
 ----
 
-. After the command has completed and all pods have been removed from the `knative-serving` namespace, delete the namespace by using the command: 
+. Once the command has completed and all pods have been removed from the `knative-serving` namespace, delete the namespace:
```
Comment (jboxman) [inline modules/serverless-uninstalling-knative-serving.adoc]: s/namespace, delete the namespace:/namespace, delete the namespace by entering the following command/

File: modules/serverless-uninstalling-knative-serving.adoc [hunk 4]
```diff
@@ -1,27 +1,22 @@
...
-
-.Prerequisite
-
-* Installed Knative Serving
+To uninstall Knative Serving, you must remove its custom resource and delete the `knative-serving` namespace.
 
 .Procedure
 
-. To remove Knative Serving, use the following command:
+. Remove Knative Serving:
```
Comment (jboxman) [inline modules/serverless-uninstalling-knative-serving.adoc]: Per IBM,

_To remove Knative Service, enter the following command:_

**Review by jboxman — CHANGES_REQUESTED** [review]
I'm still mentally working through "To ... enter the following command", but based on IBM that is the approach.

Otherwise, some comments and LGTM.

File: modules/serverless-uninstalling-knative-serving.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/serverless-uninstalling-knative-serving.adoc]: Ack, thanks!

**Comment by abrennan89** [comment]
This PR applies for OCP 4.3 and any subsequent releases (4.4, 4.5, afaik)

**Review by jboxman — APPROVED** [review]

**Pull request merged**